ginivision/src/main/java/net/gini/android/vision/internal/camera/photo/MutablePhoto.java

```diff
@@ -0,0 +1,268 @@
+package net.gini.android.vision.internal.camera.photo;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+import android.support.annotation.NonNull;
+import android.support.annotation.VisibleForTesting;
+
+import net.gini.android.vision.document.ImageDocument;
+
+import org.apache.commons.imaging.ImageReadException;
+import org.apache.commons.imaging.ImageWriteException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.UUID;
+
+/**
+ * @exclude
+ */
+class MutablePhoto extends ImmutablePhoto implements Parcelable {
```

Comment:
Contains basically the same logic as the old `Photo` class and is used for jpegs.